scripts/bin/base_nas_fanout_init.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
This file is removed in https://github.com/open-switch/opx-nas-interface/commit/78168e4e69fc99ca635dea9962d453a886d1e94c . Please don't bring this in.

Reply:
Please address this.